### ISSUE TITLE
Remove Heap code

### ISSUE THREAD

**Issue by PeterBreen** [DemocracyLab/CivicTechExchange#1048]
DemocracyLab used to use Heap for analytics, but no longer - so all Heap-related code should be removed.

 - remove all instances/uses of of  [HEAP_ANALYTICS_ID](https://github.com/search?q=repo%3ADemocracyLab%2FCivicTechExchange+HEAP_ANALYTICS_ID&type=code)  
 - remove heapApi.js from 'common/components/utils'
 - remove all uses of the [metrics](https://github.com/search?q=repo%3ADemocracyLab%2FCivicTechExchange%20metrics.&type=code) utility functions in common/components/utils/metrics.js
 - remove common/components/utils/metrics.js

Then, test the website to make sure it still renders all pages (smoke test). If the site renders all pages, no heap-related errors are in console, then this is done.

Note for code reviewer: After this code is deployed, remove the HEAP_ANALYTICS_ID key on the server, prod and all test environments.

**Comment by sp6370** [comment]
Hi @PeterBreen!
Can I pick this up?

**Comment by PeterBreen** [comment]
Yes, please! I've assigned you. Feel free to reach out on github or slack if you've got any questions.

**Comment by sp6370** [comment]
@PeterBreen Apologies for the delay. I have opened a new pull request,
